test/shared/configUtils.test.ts

```diff
@@ -7,68 +7,94 @@
 
 import { expect } from 'chai';
 import { Workspace } from '@lwc/lwc-dev-server';
-import { Config, ConfigAggregator } from '@salesforce/core';
+import { Config, ConfigAggregator, Connection } from '@salesforce/core';
 import { TestContext } from '@salesforce/core/testSetup';
 import { CryptoUtils } from '@salesforce/lwc-dev-mobile-core';
-import { ConfigUtils } from '../../src/shared/configUtils.js';
+import { ConfigUtils, LocalWebServerIdentityData, IdentityTokenService } from '../../src/shared/configUtils.js';
 import { ConfigVars } from '../../src/configMeta.js';
 
 describe('configUtils', () => {
   const $$ = new TestContext();
+  const fakeIdentityToken = 'PFT1vw8v65aXd2b9HFvZ3Zu4OcKZwjI60bq7BEjj5k4=';
+  const username = 'SalesforceDeveloper';
+  const fakeEntityId = 'entityId';
+
+  class TestIdentityTokenService implements IdentityTokenService {
+    // eslint-disable-next-line class-methods-use-this, @typescript-eslint/no-unused-vars
```

Comment:
Ugh. I was going to code up a nit suggestion for doing away with this warning, only to unearth no less than three other unavoidable warnings as I tried to write a solution. Our guardrails for opinionated TS idioms are...a lot.